src/components/agent/validators.ts

```
@@ -54,12 +54,12 @@ export const validateSupportRequest = () =>
   checkSchema({
     bot_phone: {
       in: ["body"],
-      isString: true,
       optional: true,
+      isString: true,
       trim: true,
       custom: {
         options: (value: string) =>
-          value.length > 0 ? value.length > 10 && !isNaN(Number(value)) : true,
+          value.length > 0 ? value.length >= 10 && !isNaN(Number(value)) : true,
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)